src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -82,6 +83,10 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader->load('fragment_renderer.xml');
 
+        if (class_exists(Application::class)) {
```

Comment:
Sorry but I don't understand. We are checking a class existence here, `FileExistenceResource` is about the filesystem, right?